File: Lab2/lab2/src/main/java/org/example/MatrixPrinter.java
package org.example;

public class MatrixPrinter {
    public void Print(float[][] matrix)
    {
        for (float[] row : matrix) {
            for (int j = 0; j < matrix[0].length; j++) {
                System.out.print(row[j] + " ");
            }
            System.out.println();
        }
    }
}

File: Lab8/MatrixMultiplicationWebApp/MatrixMultiplicationWebApp.Client/MatrixPrinter.cs
namespace Client;

public static class MatrixPrinter
{
    public static void Print(int[][] matrix)
    {
        foreach (var row in matrix)
        {
            foreach (var element in row)
            {
                Console.Write($"{element} ");
            }
            Console.WriteLine();
        }
    }
}